openedx/features/caliper_tracking/base_transformer.py

```diff
@@ -23,28 +23,22 @@ def __init__(self, event):
         """
         self.event = event
         self.caliper_event = dict()
-        self.add_generic_fields()
-        self.add_actor_info()
+        self.__add_generic_fields()
+        self.__add_actor_info()
+        self.__add_referrer()
+        self.__add_extensions()
 
-    def add_generic_fields(self):
+    def __add_generic_fields(self):
```

Comment:
What is the use of double underscores here?

Reply:
Indicates private class methods.